src-plugins/manualRegistration/manualRegistration.cpp

```diff
@@ -31,6 +31,13 @@
 #include <vector>
 #include <iostream>
 #include <vtkPointHandleRepresentation2D.h>
+#include <rpiCommonTools.hxx>
```

Comment:
Can you sort the includes? On Qtcreator you can automatically add sorted includes but the other have to be sorted.